docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -4,3 +4,12 @@ This guide contains instructions to upgrade from version v8.0.0 to Unreleased.
 
 **Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
+
+## Infrastructure
+- update your `docker-compose.yml` to use different image for `selenium-server` service ([#1251](https://github.com/shopsys/shopsys/pull/1251))
+    ```diff
+    selenium-server:
+    -    selenium/standalone-chrome:3.11
+    +    image: selenium/standalone-chrome-debug:3.11
+    ```
```

Comment:
```suggestion
    ```
    - now you can [watch what's going on in the Selenium browser](/docs/introduction/running-acceptance-tests.md#how-to-watch-what-is-going-on-in-the-selenium-browser) during the execution of acceptance tests
```